components/centraldashboard/frontend/index/styles.css

```diff
@@ -0,0 +1,224 @@
+    -webkit-align-items: center;
+        -ms-flex-align: center;
+            align-items: center;
+    -webkit-justify-content: center;
+        -ms-flex-pack: center;
+            justify-content: center;
+  }
+  .mdl-demo section > .section__play-btn {
+    min-height: 200px;
+  }
+  .mdl-demo section > .section__input {
+    color: #00BCD4;
+  }
+  .mdl-demo section > header > .material-icons {
+    font-size: 3rem;
+  }
+  .mdl-demo section > button {
+    position: absolute;
+    z-index: 99;
+    top: 8px;
+    right: 8px;
+  }
+  .mdl-demo section .section__circle {
+    display: -webkit-flex;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-align-items: center;
+        -ms-flex-align: center;
+            align-items: center;
+    -webkit-justify-content: flex-start;
+        -ms-flex-pack: start;
+            justify-content: flex-start;
+    -webkit-flex-grow: 0;
+        -ms-flex-positive: 0;
+            flex-grow: 0;
+    -webkit-flex-shrink: 1;
+        -ms-flex-negative: 1;
+            flex-shrink: 1;
+  }
+  .mdl-demo section .section__text {
+    -webkit-flex-grow: 1;
+        -ms-flex-positive: 1;
+            flex-grow: 1;
+    -webkit-flex-shrink: 0;
+        -ms-flex-negative: 0;
+            flex-shrink: 0;
+    padding-top: 8px;
+  }
+  .mdl-demo section .section__text h5 {
+    font-size: inherit;
+    margin: 0;
+    margin-bottom: 0.5em;
+  }
+  .mdl-demo section .section__text a {
+    text-decoration: none;
+  }
+  .mdl-demo section .section__circle-container > .section__circle-container__circle {
+    width: 64px;
+    height: 64px;
+    border-radius: 32px;
+    margin: 8px 0;
+  }
+  .mdl-demo .mdl-layout__tab .section__center {
+    margin: 50px;
+    align-items: center;
+  }
+  .mdl-demo section.section--footer .section__circle--big {
+    width: 100px;
+    height: 100px;
+    border-radius: 50px;
+    margin: 8px 32px;
+  }
+  .mdl-demo .is-small-screen section.section--footer .section__circle--big {
+    width: 50px;
+    height: 50px;
+    border-radius: 25px;
+    margin: 8px 16px;
+  }
+  .mdl-demo section.section--footer {
+    padding: 64px 0;
+    margin: 0 -8px -8px -8px;
+  }
+  .mdl-demo section.section--center .section__text:not(:last-child) {
+    border-bottom: 1px solid rgba(0,0,0,.13);
+  }
+
+  .mdl-demo .mdl-layout__tab-panel {
+    background-color: white;
+    align-self: center;
+  }
+  .mdl-demo .toc {
+    border-left: 4px solid #C1EEF4;
+    margin: 24px;
+    padding: 0;
+    padding-left: 8px;
+    display: -webkit-flex;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-flex-direction: column;
+        -ms-flex-direction: column;
+            flex-direction: column;
+  }
+  .mdl-demo .toc h4 {
+    font-size: 0.9rem;
+    margin-top: 0;
+  }
+  .mdl-demo .toc a {
+    color: #4DD0E1;
+    text-decoration: none;
+    font-size: 16px;
+    line-height: 28px;
+    display: block;
+  }
+  .mdl-demo .mdl-menu__container {
+    z-index: 99;
+  }
+  
```

Comment:
ditto

Reply:
Edited out the newline. PTAL.

Reply:
Sorry, do you add a whitespace at line 224?
<img width="356" alt="screen shot 2018-02-01 at 10 23 01" src="https://user-images.githubusercontent.com/5319646/35657941-02839276-073a-11e8-9b1f-625524c1c020.png">